trino-aws-proxy-glue/src/test/java/io/trino/aws/proxy/glue/TestingGlueRequestHandler.java

```diff
@@ -94,6 +95,15 @@ public GlueResponse handleRequest(ParsedGlueRequest request, SigningMetadata sig
                 yield CreateDatabaseResponse.builder().build();
             }
 
+            case CreateTableRequest tableRequest -> {
```

Comment:
Since you are adding a good number of cases to this, do you think its working making `TestingGlueRequestHandler` a wrapper that takes a delegate set by the current test and forwards to that?

The asserts for requests would live next to the actual call

Reply:
I  don't see many more of these actually. Both of these cases test all the serialization that we need. The only one I missed last time were collections (e.g. generics in general). So, let's leave it like this for now.